src/form/birth/optional-fields.ts

```diff
@@ -61,12 +61,8 @@ export const weightAtBirth: SerializedFormField = {
       parameters: [0, 6]
     }
   ],
-  postfix: 'Kg',
-  mapping: getFieldMapping(
-    'weightAtBirth',
-    certificateHandlebars.weightAtBirth
-  ),
-  inputFieldWidth: '78px'
+  postfix: 'kilograms (kg)',
```

Comment:
we might need to translate this via `formMessageDescriptors` ?

Reply:
currently we only support strings for this prop, adding intl message support could be a good addition